filesystem/btrfs_stream.ksy

```diff
@@ -0,0 +1,151 @@
+meta:
+  id: btrfs_stream
+  application: Btrfs
+  file-extension: btrfs-stream
```

Comment:
I couldn't find mentions of this exact file extension to be used anywhere in the documentation referenced. It's not a mandatory field, and we should not be inventing extensions if they don't exist in reality. If that's the case, shall we remove it?

```suggestion
```

Reply:
File extension removed